src/components/projectInfo/projectInfo.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import {  Row , Col , Image} from 'react-bootstrap'
+
+const ProjectInfo = () => {
+	return (
+		<Row className=" no-gutters bg-light position-relative align-items-center">
+			<Col className="col-md-2"></Col>
+			<Col className="col-md-4 p-4 p-md-4">
+				<Image src="https://www.polska.travel/images/pl-PL/glowne-miasta/gdansk/gdansk_motlawa_1170.jpg" className="bd-placeholder-img bd-placeholder-img-lg featurette-image img-fluid mx-auto" alt="..." />
```

Comment:
jeśli korzystamy z dostępnych elementów html, to piszemy tagi z małych liter np. `div`, `a`, `p`, `ul`, `li`, `img` - czy pisząc `Image` chodziło o tag `img` czy komponent `Image`? Jeśli komponent to brakuje importu  :)

Reply:
Image jest importowane z 'react-bootstrap'